about.html

```diff
@@ -0,0 +1,222 @@
+                <div class="kts flex">
+                    <div class="kts-headings flex">
+                        <img src="images/why-kts-01.png" alt="Elevate Icon" class="kts-icon">
+                        <h3 class="kts-title secondary-color">Elevate</h3>
+                    </div>
+                    <p class="kts-description bold">Your time is precious so use it wisely!
+                        At DTS22, you’ll find all your tech needs under one roof. Learn how to
+                        work smarter, not harder. Sharpen your instincts to ensure you stay one
+                        step ahead of the competition. Expand your knowledge with insights from
+                        inspirational industry leaders and upskill with exposure to the latest
+                        cutting-edge technology.</p>
+                </div>
+            </div>
+            <!--Why KDS 02- Integrate-->
+            <div class="kts-wrapper" data-aos="slide-up" data-aos-delay="50" data-aos-duration="1000">
+                <div class="kts flex">
+                    <div class="kts-headings flex">
+                        <img src="images/why-kts-02.png"  alt="Integrate Icon" class="kts-icon">
+                        <h3 class="kts-title secondary-color">Integrate</h3>
+                    </div>
+                    <p class="kts-description bold">Unique opportunities to meet with
+                        C-Suite attendees and investors. Connect with responsive tech experts
+                        from all over the world. Share ideas with the best influencers in their
+                        fields and participate in thought-provoking panel sessions. Exchange
+                        knowledge and strategies with an energised global community.</p>
+                </div>
+            </div>
+            <!--Why KDS 02- Accelerate-->
+            <div class="kts-wrapper" data-aos="slide-up" data-aos-delay="50" data-aos-duration="1000">
+                <div class="kts flex">
+                    <div class="kts-headings flex">
+                        <img src="images/why-kts-03.png"  alt="Accelerate Icon" class="kts-icon">
+                        <h3 class="kts-title secondary-color">Accelerate</h3>
+                    </div>
+                    <p class="kts-description bold">Discover new ways to build your brand and
+                        get immediate traction. Amplify and scale your business for a bigger audience.
+                        Connect and grow your database with an engaged audience of over 10,000 informed
+                        delegate. Get on the investor track with unique access to the decision-makers
+                        at specially curated events. Step-up your engagement with global thought-leaders.</p>
+                </div>
+            </div>
+        </div>
+        <div class="button">
+            <a href="#" class="join-summit white">Join Kada Tech Summit 2022</a>
+        </div>
+    </section>
```

Comment:
- [ ] While you have made a great effort in creating and styling this section, kindly note that it is not part of the project according to the provided Behance design and should therefore not be included in the project. Kindly remove this section and in its place put the design that is required as per the Behance design.

**This is the desired section. You are allowed to personalize it to your wish but not entirely alter the design**

![image](https://user-images.githubusercontent.com/86318284/181570585-aa7e2921-8c27-47f4-98be-2c5fe742b090.png)

Reply:
Well Noted! I just removed both the Organizers section and the Event section. And I replaced them with a section about the event logo as contained in the design guideline.